pkg/crc/machine/console.go

```diff
@@ -17,9 +16,9 @@ func (client *client) GetConsoleURL() (*types.ConsoleResult, error) {
 	}
 	defer vm.Close()
 
-	vmState, err := vm.Driver.GetState()
+	vmState, err := vm.State()
 	if err != nil {
-		return nil, errors.Wrap(err, "Error getting the state of the virtual machine")
+		return nil, errors.Wrap(err, "Error getting the state for host")
```

Comment:
We are getting state of the VM so why error message is changed? I think initial one is more correct.

Reply:
Some small rebase issue, I'll fix it, thanks for catching it!